cookbooks/allken_et_al_2012_rift_interaction/allken.prm

```diff
@@ -0,0 +1,168 @@
+#  Global parameters
+set Dimension                              = 3
+set Start time                             = 0
+set End time                               = 5e6
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = iterated Stokes
+set Nonlinear solver tolerance             = 1e-3
+set Max nonlinear iterations               = 25
+set Output directory                       = output_allken_etal_2012
+set Pressure normalization                 = no
+set CFL number                             = .25
+
+# Solver settings
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Number of cheap Stokes solver steps = 0
+  end
+end
+
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 10
+    set Y repetitions = 10
+    set Z repetitions = 2
+    set X extent      = 210e3
+    set Y extent      = 210e3
+    set Z extent      = 30e3
+  end
+end
+
+# Mesh refinement specifications.
+subsection Mesh refinement
+  set Initial adaptive refinement        = 2
+  set Initial global refinement          = 2
+  set Time steps between mesh refinement = 1
+  set Refinement fraction                = 0.5
+  set Strategy                           = strain rate
+  set Coarsening fraction                = 0
+  #set Run postprocessors on initial refinement = true
+end
+
+# Element types
+subsection Discretization
+  set Composition polynomial degree     = 2
+  set Stokes velocity polynomial degree = 2
+  set Temperature polynomial degree     = 1
+end
+
+subsection Mesh deformation
+  set Mesh deformation boundary indicators = top: free surface
+  set Additional tangential mesh velocity boundary indicators = left,right
+  subsection Free surface
+    set Surface velocity projection = normal
+  end
+end
+
+# Velocity boundary conditions
+# The imposed velocity produces a background strain-rate of 2e-15.
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = 2,3,4 #bottom,
```

Comment:
Please use the names. I think this is `right, top, bottom`?

Reply:
done